src/content/docs/concepts/programs.mdx

```diff
@@ -6,7 +6,7 @@ import { FileTree } from "@astrojs/starlight/components";
 
 Membrane is similar to an operating system; you deploy programs to the Membrane engine, and it takes care of persistence, communication, logging, resources, etc.
 
-A Membrane _program_ is TypeScript or JavaScript code that runs in Membrane's serverless cloud runtime. Unlike traditional serverless runtimes, it is [stateful](/features/state/) and includes multiple handlers instead of a single function.
+A Membrane _program_ is TypeScript or JavaScript code that runs our cloud runtime. Unlike traditional serverless runtimes, it is [stateful](/features/state/) and includes multiple handlers instead of a single function.
```

Comment:
```suggestion
A Membrane _program_ is TypeScript or JavaScript code that runs in our cloud runtime. Unlike traditional serverless runtimes, it is [stateful](/features/state/) and includes multiple handlers instead of a single function.
```